src/method/technology.tex

```diff
@@ -0,0 +1,100 @@
+\section{Technology}\label{sec:technology}
+
+Throughout the development of the project, a number of different tools and publicly available packages were used to ease
+the development process.
+These different technologies were chosen based on the team's needs and the requirements of the project.
+The following sections will describe the different technologies used in the project.
+
+\subsection{Programming stack}\label{subsec:programming}
+
+As the chess game would be hosted on a website, and will require multiple users to interact with each other, the team
+decided to use TypeScript~\cite{typescript} as the main programming language.
+TypeScript is a superset of JavaScript, with the main difference being that TypeScript is statically typed, while
+JavaScript is dynamically typed.
+Typescript can be very useful in larger projects, as it can help catch errors before the code is run.
+While many in the team did not have any prior experience with TypeScript, in comparison to JavaScript, they quickly
+learned the basics of the language and were able to use it effectively.
+
+The frontend of the project was built with React~\cite{react}, which is a JavaScript / TypeScript library for building
+user interfaces.
+It is developed and maintained by Meta, and is used by many large companies.
+React is focused on mobile applications first and foremost, but the team's focus was on the desktop experience.
+
+% textidote: ignore begin % due to node.js and next.js being identified as an url
+The backend was built using Node.js~\cite{node.js}, which allows the team to build the server using JavaScript.
+Node.js is a popular choice for backend development, as it is fast and scalable.
+As the team is using React for client-side rendering, they decided to also incorporate Next.js~\cite{next.js} to handle
+the server-side rendering.
+Next.js is a React framework running on top of Node.js, which makes it easier to build the frontend and backend
+together.
+% textidote: ignore end
+
+As Stockfish~\cite{stockfish} is built on C++, the team had to find a way to interact with it.
+To do this, the team created a REST API using Python, as it was deemed the easiest to implement using
+py-stockfish~\cite{py-stockfish} as a wrapper.
```

Comment:
Again a little too storytelling-esque in my opinion. Also remember to use present tense.